internal/characters/diluc/skill.go

```diff
@@ -76,6 +75,22 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 				return m, true
 			},
 		})
+
+		mAtkSpd := make([]float64, attributes.EndStatType)
+		mAtkSpd[attributes.AtkSpd] = 0.3
+		c.AddStatMod(character.StatMod{
+			Base:         modifier.NewBaseWithHitlag("diluc-c6-speed", 360),
+			AffectedStat: attributes.AtkSpd,
+			Amount: func() ([]float64, bool) {
+				if c.Core.Player.CurrentState() != action.NormalAttackState {
+					return nil, false
+				}
+				if count > 1 {
+					return nil, false
+				}
+				return mAtkSpd, true
```

Comment:
The instance of `count` isn't shared between the two mods so you'd have to make it into a char variable if you want to share it.